src/main/resources/assets/jade/lang/uk_ua.json

```diff
@@ -1,191 +1,387 @@
+	"config.jade.plugin_minecraft.redstone": "Інформація про редстоун",
+	"config.jade.plugin_minecraft.jukebox": "Запис програвача",
 	"config.jade.plugin_minecraft.villager_profession": "Професія селянина",
-	"config.jade.plugin_minecraft.chiseled_bookshelf": "Різьблена книжкова полиця",
 	"jade.spawner": "%s (%s)",
-	"toast.jade.toggle_hint.1": "Jade закрито",
-	"toast.jade.toggle_hint.2": "Натисніть %s, щоб відкрити знову",
+	"jade.trial_spawner_cd": "Перезарядка: %s",
+	"config.jade.plugin_minecraft.chiseled_bookshelf": "Різьблена книжкова полиця",
+	"config.jade.plugin_minecraft.zombie_villager": "Перетворення зомбі-селянина",
+	"jade.zombieConversion.time": "Час перетворення: %s",
+	"config.jade.plugin_minecraft.waxed": "Стан вощення",
+	"config.jade.plugin_minecraft.waxed_desc": "Показати стан вощеного блоку, наприклад, вощений знак.",
+	"jade.waxed": "Вощений %s",
+	"config.jade.accessibility_plugin": "Плаґін доступності",
+	"config.jade.accessibility_plugin_desc": "Увімкніть функції доступності для відображення та розповіді більш детальної інформації. Потім ви можете керувати більш детальною інформацією у налаштуваннях плаґіна.",
+	"config.jade.plugin_jade_access": "Доступність",
+	"config.jade.plugin_jade_access.sign": "Зміст таблички",
+	"jade.access.sign.front": "Front:",
+	"jade.access.sign.back": "Back:",
+	"jade.access.sign.line1": "Перший рядок: %s",
+	"jade.access.sign.line2": "Другий рядок: %s",
+	"jade.access.sign.line3": "Третій рядок: %s",
+	"jade.access.sign.line4": "Четвертий рядок: %s",
+	"config.jade.plugin_jade_access.block": "Деталі блоку",
+	"jade.access.block.door_open": "%s",
+	"jade.access.block.door_closed": "Закрито %s",
+	"jade.access.block.waterlogged": "Затоплено %s",
+	"jade.access.block.lit": "Запалено %s",
+	"jade.access.block.inverted": "Інвертований %s",
+	"jade.access.block.eye": "%s з оком",
+	"jade.access.block.ominous": "Зловісний %s",
+	"jade.access.block.active": "Активний %s",
+	"jade.access.block.hydrated": "Зволожений %s",
+	"jade.access.block.locked": "Замкнуто %s",
+	"jade.access.block.extended": "Розширений %s",
+	"jade.access.block.amount": "Кількість: %s",
+	"jade.access.block.level": "Рівень: %s",
+	"jade.access.block.layers": "Шари: %s",
+	"jade.access.block.bites": "Укуси: %s",
+	"jade.access.block.facing": "Сторона: %s",
+	"jade.access.block.redstone_wire": "Форма: %s",
+	"jade.access.block.redstone_wire.dot": "Форма: Точка",
+	"config.jade.plugin_jade_access.entity": "Деталі сутностей",
+	"jade.access.creeper.powered": "Заряджений %s",
+	"jade.access.zombie_villager.curing": "Вилікування %s",
+	"jade.access.goat.hornless": "Безрогий %s",
+	"jade.access.wither.powered": "Посилений %s",
+	"jade.access.entity.sheared": "Стрижений %s",
+	"jade.access.entity.saddled": "Осідланий %s",
+	"jade.access.entity.baby": "Дитинча %s",
+	"jade.access.entity.angry": "Злий %s",
+	"jade.access.bee.nectar": "Нектар %s",
+	"jade.access.slime.size": "{1,choice,1#Small|2≤Medium|4≤Large|8≤Huge} {0}",
```

Comment:
please translate Small, Medium, Large and Huge

Reply:
It's done